lib/shopify_transporter/exporters/magento/soap.rb

```diff
@@ -7,21 +7,55 @@ module ShopifyTransporter
   module Exporters
     module Magento
       class Soap
-        def initialize(hostname: '', username: '', api_key: '')
+        RETRY_SLEEP_TIME = 0.1
+        MAX_RETRIES = 4
+
+        def initialize(hostname: '', username: '', api_key: '', batch_config:)
           @hostname = hostname
           @username = username
           @api_key = api_key
+          @batch_config = batch_config
+        end
+
+        def call(method, params)
```

Comment:
To be consistent with the interface of `call_in_batches`, should we make this also `method:, params:`?

Reply:
Or conversely changing `call_in_batches` _not_ to use named parameters?  Most methods are not using names params, so perhaps that should be the convention we go with.

Reply:
Done!